backend/apid/actions/delete_events_by_entity.go

```diff
@@ -11,8 +11,7 @@ import (
 )
 
 type EntityDeleter struct {
-	EntityStore store.EntityStore
-	EventStore  store.EventStore
```

Comment:
I'm sorry to say, this will break the EntityDeleter when the postgres event store is used. The separate EventStore needs to stay until we have a wholly swappable Store.

Reply:
I didn't realize this had implications on postgres, good catch! I've added it back.